### ISSUE TITLE
trying with command but nothing retrieving in csv

### ISSUE THREAD

**Issue by Kamal-Sonani** [0xRyuk/JD-Scrape#1]
response code 200 & print this in terminal

`[15:38:15] [INFO] Total 0 Services found.`
`[15:38:15] [INFO] Results saved in 'justdial.csv'`
`[15:38:15] [INFO] Scraping finished in 2.644 second(s)`

**Comment by 0xRyuk** [comment]
Yes, this is happening because the front-end code of the Justdial website has been updated, which has resulted in changes to the elements and their locations. If I have enough time, I will rebuild the logic again.
